snuba/query/parser/expressions.py

```diff
@@ -52,6 +58,32 @@ def visit_function_name(self, node: Node, visited_children: Iterable[Any]) -> st
     def visit_column_name(self, node: Node, visited_children: Iterable[Any]) -> Column:
         return Column(None, None, node.text)
 
+    def visit_empty(self, node, children):
```

Comment:
Please provide type hints and return value for all these methods.

Reply:
Added.